### ISSUE TITLE
Building `gradle/gradle` fails with Error:Kotlin: Output directory not specified for Module 'org.gradle.dependencyManagement.integTest' production

### ISSUE THREAD

**Issue by wolfs** [gradle/gradle#8100]
When trying to run a test from IDEA, I get the following error:
```
Error:Kotlin: Output directory not specified for Module 'org.gradle.dependencyManagement.integTest' production
```
![image](https://user-images.githubusercontent.com/423186/50329778-f7f5ea80-04f8-11e9-98f2-9206ed015ec3.png)

I was trying to run `ArtifactTransformIntegrationTest` and just synched the project.

I restarted IDEA, and still have the same problem.

**Comment by wolfs** [comment]
I manually added the output path, and now I can run the test:
![image](https://user-images.githubusercontent.com/423186/50330061-1c9e9200-04fa-11e9-9f30-9f545946275b.png)

Doing a Gradle sync removes the output path again.
Building and running the test works now, even though the configuration is the same.

**Comment by wolfs** [comment]
Got the error again, this time for `org.gradle.core.integTest`, trying to run `TaskFilePropertiesIntegrationTest`. After thinking a little bit about it, it seems like the actual output path is wrong - why do the `integTest` classes go into the `test` output path? I think the test output path should be `out/integTest/classes`.

The `testFixtures` also go into `out/testFixtures/classes` (as output path).

This happened to me after a sync of the Gradle configuration, so maybe it is related.

**Comment by chusmc** [comment]
I have the same issue

I configured using gradle a new sourceSet called intTest and a new test task called also intTest
I used the idea plugin in gradle to add the source dirs to the testSourceDirs.

Everything looks correct, and intelliJ created a separate module per source set, so I have intTest and test as modules. 
The output path of the intTest module is empty, but the test output path is pointing to the test classes output
When I try to run one of the tests in the intTest module I get this error:

Error:Kotlin: Output directory not specified for Module 'intTest' production

Is there anyway to mark that intTest module as non production?
